test/remote/gateways/remote_commerce_hub_test.rb

```diff
@@ -404,4 +404,11 @@ def test_successful_purchase_with_encrypted_credit_card
     assert_success response
     assert_equal 'Approved', response.message
   end
+
+  def test_client_request_id_is_uuidv4
+    client_request_id = SecureRandom.uuid
+    response = @gateway.purchase(@amount, @credit_card, @options.merge({ client_request_id: client_request_id }))
```

Comment:
@sinourain can you add `assert_success response` so that we can confirm that tests passes